python/packages/sdk/serverless_sdk/lib/captured_event.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
```

Comment:
Can we have some tests for this file and `error_captured_event` (?)

Reply:
Good point, I've added tests 👍